src/config/mod.rs

```diff
@@ -330,21 +331,17 @@ impl Config {
         let proxy = self
             .proxy
             .as_ref()
-            .map(|v| v.to_string())
-            .unwrap_or("-".into());
+            .map_or_else(|| String::from("-"), std::string::ToString::to_string);
         let temperature = self
             .temperature
-            .map(|v| v.to_string())
-            .unwrap_or("-".into());
+            .map_or_else(|| String::from("-"), |v| v.to_string());
         let (api_key, organization_id) = self.get_api_key();
         let api_key = mask_text(&api_key, 3, 4);
-        let organization_id = organization_id
-            .map(|v| mask_text(&v, 3, 4))
-            .unwrap_or("-".into());
+        let organization_id = organization_id.map_or_else(|| "-".into(), |v| mask_text(&v, 3, 4));
```

Comment:
Purely stylistic, but this is more readable and idiomatic.

https://rust-lang.github.io/rust-clippy/master/index.html#/map_unwrap_or